azure-managed-lustre/use-csi-driver-kubernetes.md

```diff
@@ -13,129 +13,143 @@ ms.reviewer: brianl
+1. [Create and configure a persistent volume](#create-and-configure-a-persistent-volume).
 
-- With this option, you don't have to create or size the virtual network and subnet. `kubenet` automatically creates an appropriately sized subnet for your AKS cluster.
+1. [Check the installation](#check-the-installation) by optionally using an echo pod to confirm the driver is working.
 
-- The AKS virtual network is created in a separate resource group from the AKS cluster's resource group. Look for a resource group with a name that starts with **MC_** and includes the name of your AKS cluster. This resource group contains the AKS virtual network and related components.
+The following sections describe each task in greater detail.
 
-  For more information, see [Kubenet (basic) networking](/azure/aks/concepts-network#kubenet-basic-networking) in [Network concepts for applications in Azure Kubernetes Service (AKS)](/azure/aks/concepts-network).
+## Create an Azure Managed Lustre file system
 
-- To learn how to connect the two networks, see [Virtual network peering](/azure/virtual-network/virtual-network-peering-overview).
+If you haven't already created your Azure Managed Lustre file system cluster, create the cluster now. For instructions, see [Create an Azure Managed Lustre file system in the Azure portal](create-file-system-portal.md). Currently, the driver can only be used with a existing Azure Managed Lustre file system.
 
-## Create an Azure Managed Lustre file system
+## Create an AKS Cluster
 
-If you haven't already created your Azure Managed Lustre File System cluster, create the cluster now. For instructions, see [Create an Azure Managed Lustre file system in the Azure portal](create-file-system-portal.md). Currently, the driver can only be used with a pre-existing Azure Managed Lustre system.
+If you haven't already created your AKS cluster, create a cluster deployment. See [Deploy an Azure Kubernetes Service (AKS) cluster](/azure/aks/learn/quick-kubernetes-deploy-portal).
 
-## Connect a client to your Azure Managed Lustre file system
+## Create virtual network peering
 
-Do the following steps to install a Lustre client, and mount the file system:
+> [!NOTE]
+> Skip this network peering step if you installed AKS into a subnet on the Azure Managed Lustre virtual network.
 
-1. Review the steps for installing the client software and mounting the system in [Connect to clients](connect-clients.md).
+The AKS virtual network is created in a separate resource group from the AKS cluster's resource group. You can find the name of this resource group by going to your AKS cluster in the Azure Portal choosing the **Properties** blade and finding the **Infrastructure** resource group.  This resource group contains the virtual network that needs to be paired with the Azure Managed Lustre virtual network. It matches the pattern **MC\_&lt;aks-rg-name&rt;\_&lt;aks-cluster-name&rt;\_&lt;region&rt;**.
```

Comment:
```suggestion
The AKS virtual network is created in a separate resource group from the AKS cluster's resource group. You can find the name of this resource group by going to your AKS cluster in the Azure Portal choosing the **Properties** blade and finding the **Infrastructure** resource group.  This resource group contains the virtual network that needs to be paired with the Azure Managed Lustre virtual network. It matches the pattern **MC\_\<aks-rg-name\>\_\<aks-cluster-name\>\_\<region\>**.
```
The right side angle brackets weren't displaying correctly so I've made changes to fix them... but @jeffbearer - please check that this is the correct pattern: 

MC\_\<aks-rg-name\>\_\<aks-cluster-name\>\_\<region\>